src/Sulu/Bundle/AdminBundle/Resources/js/components/Tabs/README.md

```diff
@@ -0,0 +1,58 @@
+With `Tabs` you can easily switch between different views. A `Tab` can contain basic HTML, Text and also other React
+components.
+
+```
+initialState = {
+    value: '1',
+};
+
+const handleChange = (value) => {
+    setState({
+        value: value
+    });
+};
+
+const styles = {
+    tabContent: {
+        padding: 20,
+        lineHeight: 1.5,
+    },
+};
+
+<Tabs value={state.value} onChange={handleChange}>
+    <Tabs.Tab label="Cheeseburger" value="1">
```

Comment:
I would say that we could use the index of the tab instead of passing a value manually. [React Tabs](https://github.com/reactjs/react-tabs#controlled-mode) is doing the same. /cc @chirimoya